# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../resynthesizer/install.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- resynthesizer-0.14/Makefile_orig	2006-08-26 18:20:57.000000000 +0100
+++ resynthesizer-0.14/Makefile	2006-08-26 18:21:11.000000000 +0100
@@ -17,9 +17,9 @@
 	@echo 
 
 install: resynth smart-enlarge.scm smart-remove.scm
-	$(GIMPTOOL) --install-bin resynth
-	$(GIMPTOOL) --install-script smart-enlarge.scm
-	$(GIMPTOOL) --install-script smart-remove.scm
+	$(GIMPTOOL) --install-admin-bin resynth
+	$(GIMPTOOL) --install-admin-script smart-enlarge.scm
+	$(GIMPTOOL) --install-admin-script smart-remove.scm
 	@echo
 	@echo After restarting the Gimp you should find the
 	@echo following items in the pop-up image menu:
